app.go

```diff
@@ -240,6 +246,31 @@ func executeQuote(cmd command) bool {
 		return false
 	}
 
+	//Check auto buy sell
+
+	for _, v := range (*autoBuyRequestStore)[stock] {
```

Comment:
Gonna shelve this for now, since future implementation will result in a major refactor anyway.

There are two ways to view this

1. This operation is expensive so we don't want to run it unless it's necessary. So, we should only do it on cache misses.
1. Even if it's a cache hit we may have added new triggers that need to be checked. However, these could be checked on insertion into the store.